packages/eas-cli/src/user/fetchUser.ts

```diff
@@ -0,0 +1,38 @@
+import gql from 'graphql-tag';
+
+import { createGraphqlClient } from '../commandUtils/context/contextUtils/createGraphqlClient';
+
+export async function fetchUserAsync({
+  sessionSecret,
+}: {
+  sessionSecret: string;
+}): Promise<{ id: string; username: string }> {
+  const graphqlClient = createGraphqlClient({ accessToken: null, sessionSecret: null });
```

Comment:
can just pass in `sessionSecret` here?

Reply:
Both args are needed

Reply:
Sorry I wan't clear. I meant can we pass `sessionSecret` here instead of adding it to `fetchOptions` below?

Reply:
I passed it in as recommended and deleted the fetchOptions